### PR TITLE
Escalation engineer rotation for May 2023

### PR DESCRIPTION
**Pull request by taras-yemets**
Adds @gl-srgr and @taras-yemets to the escalation engineer roation for May 2023.

### CODE REVIEW

**Comment by Strum355** [comment]
Im not sure I'm doing this in May, thought i was down for June cc @kalanchan

**Comment by taras-yemets** [comment]
> Im not sure I'm doing this in May, thought i was down for June cc @kalanchan

I took it from [EE rotation spreadsheet](https://docs.google.com/spreadsheets/d/1U3eAhQnLaYwe8-_vE11hnLaiS638zLyDwGrzKpnxrrM/edit#gid=0) ([thread](https://sourcegraph.slack.com/archives/C03TGHH1S3V/p1681823732789139?thread_ts=1681823682.976209&cid=C03TGHH1S3V)).

cc: @Strum355

**Review by kopancek — APPROVED** [review]

File: content/departments/engineering/dev/process/escalation-engineer-rotation.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Discuss escalation engineering rotation in #escalation-engineering.
 | 2022-02-03 to 2023-03-03 | [Beatrix Woo](https://github.com/abeatrix), [Jacob Pleiness](https://github.com/jdpleiness) ,[Manuel Ucles](https://github.com/mucles)        |
 | 2022-03-03 to 2023-04-05 | ?                                                                                                                                             |
 | 2022-04-05 to 2023-05-05 | [David Veszelovszki](https://github.com/vdavid), [Milan Freml](https://github.com/kopancek), [Idan Varsano](https://github.com/varsanojidan)  |
+| 2022-05-05 to 2023-06-05 | [Noah Santschi-Cooney](https://github.com/Strum355), [Gary Lee](https://github.com/gl-srgr), [Taras Yemets](https://github.com/taras-yemets)  |
```
Comment (kalanchan) [inline content/departments/engineering/dev/process/escalation-engineer-rotation.md]:
```suggestion
| 2022-05-05 to 2023-06-05 | [Gary Lee](https://github.com/gl-srgr), [Taras Yemets](https://github.com/taras-yemets)  |
```

**Review by kalanchan — CHANGES_REQUESTED** [review]
hey @taras-yemets thanks for following up on this. We're going to move @Strum355 over to June and have you and @gl-srgr on EE rotation in May. 

This will help us spreadout the workload :)

**Pull request merged**